src/reducers/resources.js

```diff
@@ -62,17 +62,23 @@ export const setCurrentResource = (state, action) => {
 export const addSubject = (state, action) => {
   const newState = { ...state }
 
+  const subject = state.entities.subjects[action.payload.key]
   const newSubject = { ...action.payload }
   delete newSubject.properties
   delete newSubject.subjectTemplate
   newState.entities.subjects[action.payload.key] = newSubject
 
+  if (!_.isEqual(newSubject, subject)) {
+    newState.entities.subjects[newSubject.resourceKey].changed = true
```

Comment:
I think setting this value to `true` is creating a regression because when a resource template is initially loaded and nothing has changed in the state, the **Save** button (`src/components/editor/actions/SaveAndPublishButton`) is enabled when it shouldn't be. Here is a screen shot just after the abbreviated title resource template is loaded with the save button enabled (should be disabled because nothing has changed):
![Screen Shot 2020-07-15 at 4 49 40 PM](https://user-images.githubusercontent.com/71847/87607957-dc929780-c6bb-11ea-924b-a26a1a6a150b.png)

Reply:
Fixed by setting `changed` to false after resource is loaded.